master/introduction/containers/network-architecture.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Network Architecture
+---
+
+The following overview is only valid for **container** deployments:
```

Comment:
I don't think "container" deployments is necessarily something users are going to understand, especially ones who are just getting introduced to Calico.